src/main/kotlin/org/rucca/cheese/team/TeamMembershipService.kt

```
@@ -0,0 +1,630 @@
+                    id = invitationId,
+                    userId = userId,
+                    type = ApplicationType.INVITATION,
+                    status = ApplicationStatus.PENDING,
+                )
+                .orElseThrow { NotFoundError("Pending invitation for user", invitationId) }
+
+        val teamId = application.team.id!! // Get team ID
+
+        try {
+            teamService.checkTeamLockingStatus(teamId)
+        } catch (e: TeamLockedError) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)